src/ngraph/ngraph_emitter.cc

```diff
@@ -204,6 +204,21 @@ void Emitter::CreateUnaryOps() {
   // ngraph_op_funcs_["log_softmax"] = [this](const NodePtr& node){
   //   return ;
   // };
+  ngraph_op_funcs_["SoftmaxActivation"] = [this](const NodePtr& node) {
+    ngraph::AxisSet axes;
+
+    auto in_shape = op_map_[node->inputs_[0]]->get_shape();
+    auto mode = get_default(node, "mode", std::string("instance"));
+
+    if (mode == std::string("channel")) {
```

Comment:
Why the explicit use of `std::string`?

Reply:
I was getting odd compilation errors related to booleans, so I just made it explicit.